enatega-multivendor-app/src/screens/Login/Login.js

```diff
@@ -129,16 +129,16 @@ function Login(props) {
                         />
                         <FontAwesome
                           onPress={() => setShowPassword(!showPassword)}
-                          name={!showPassword ? 'eye' : 'eye-slash'}
+                          name={showPassword ? 'eye' : 'eye-slash'}
                           size={24}
                           color={
                             passwordError === null
                               ? currentTheme.black
                               : currentTheme.textErrorColor
                           }
                           style={[
-                            styles().eyeBtn,
-                            Platform.OS === 'android' && { marginTop: 14 }
+                            styles().eyeBtn
+                            // Platform.OS === 'android' && { marginTop: 14 }
```

Comment:
Have you made sure to test it on android device after change, because previously margin is added for some purpose?

Reply:
Yes I have tested on android device, I have made it vertically center so now no need of marginTop